dbms/src/Storages/KVStore/MultiRaft/RegionCFDataBase.cpp

```diff
@@ -119,9 +127,8 @@ size_t RegionCFDataBase<Trait>::calcTiKVKeyValueSize(const TiKVKey & key, const
 {
     if constexpr (std::is_same<Trait, RegionLockCFDataTrait>::value)
     {
-        std::ignore = key;
-        std::ignore = value;
-        return 0;
+        // We start to count size of Lock Cf.
+        return key.dataSize() + value.dataSize();
```

Comment:
I think mixing a logic change in a "refactor" PR is not a wise choice

Reply:
These changed are cherry-picked from another bigger PR to here in order to make these more important code changes take place.
This is not a "refactor" PR, this PR servers large txn. It is attached with some refactor though,